baremetal/metal3data_manager_test.go

```diff
@@ -155,7 +150,11 @@ var _ = Describe("Metal3Data manager", func() {
 		}),
 		Entry("requeue error", testCaseReconcile{
 			m3d: &infrav1.Metal3Data{
-				ObjectMeta: testObjectMeta,
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "abc",
```

Comment:
Based on the linked [discussion](https://github.com/metal3-io/cluster-api-provider-metal3/pull/563#discussion_r831825842) it was recommended to use a constant instead of the "abc"

Reply:
I agree. 
That will come next. For now, we are trying to have each test to have its own object so that we can override the name, namespace and uid values